src/aat/resources/features/F-1009 - Submit Case Creation Handle Case Links/CaseLinkVerifications/F-1009-VerifyCaseLinks.td.json

```diff
@@ -0,0 +1,45 @@
+{
+	"title": "should retrieve case links for a specified case reference",
+	"_guid_": "F-1009-VerifyCaseLinks",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve case links for an existing case reference",
+
+	"method": "GET",
+	"uri": "/testing-support/case-link/{cid}",
+
```

Comment:
Can we use a common TD file to cover the API signature in the common area (i.e. to be used by both F-1008 & F-1009)?